src/index.js

```diff
@@ -1,11 +1,39 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
 import App from './App';
+import rootReducer from './reducers';
+
+const defaultStore = {
+  books: [
+    {
+      id: parseInt(Math.random() * 1000, 10),
+      title: 'The Peaceful Warrior',
+      category: 'Biography',
+    },
+    {
+      id: parseInt(Math.random() * 1000, 10),
+      title: 'Harry Potter',
+      category: 'Kids',
+    },
+    {
+      id: parseInt(Math.random() * 1000, 10),
+      title: 'The Martian',
+      category: 'Sci-Fi',
+    },
+  ],
+};
+
+const store = createStore(rootReducer,
+  { books: defaultStore.books });
```

Comment:
**OPTIONAL**: Isn't `defaultStore` already and object with that shape, why get the `books` property and create a new object with the same shape instead of directly passing `defaultProps`?

Reply:
Great recommendation. Thank you!